packages/client/src/views/FieldAgentHome/FieldAgentHome.test.tsx

```diff
@@ -78,7 +78,7 @@ const countQueryGraphqlMock = {
 merge(mockUserResponse, nameObj)
 
 describe('FieldAgentHome tests', () => {
-  const { store } = createStore()
+  const { store, history } = createStore()
```

Comment:
As we have used the timeout  function now, so we don't need to change it anymore. Can you please revert it ?

Reply:
Thanks @mushrafulhoque-dsi.  Done in [this](https://github.com/opencrvs/opencrvs-core/pull/2066/commits/50c61758c1b47792680056942b7b67b250d59072) commit